designsafe/static/scripts/data-depot/components/data-depot-listing/data-depot-listing.component.js

```diff
@@ -148,6 +150,27 @@ class FilesListingCtrl {
             default:
                 return 'fa-file-o';
         }
+    }
+    isNotebook(file) {
+        const extension = file.name.split('.').pop();
+        return extension === 'ipynb';
+    }
+    hasNotebooks() {
```

Comment:
`hasNotebooks` should probably be added as a test under the `FileOperationService.getTests`. There are examples there of how to exclude certain router states without having to use `window.location`.